src/App/Form/Proposal/PublishProposalType.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Form\Proposal;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use App\Entity\Proposal;
+
+class PublishProposalType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('theme', 'entity', [
+                'class'    => 'App:Theme',
+                'property' => 'title',
+                'required' => true,
```

Comment:
I think you should then add a constraint NotNull on your `theme` property in your proposal entity.
`Required` is only to have front validation (html5).

Reply:
Yes you're right